packages/blade/src/components/Dropdown/__tests__/Dropdown.native.test.tsx

```diff
@@ -48,6 +48,24 @@ describe('<Dropdown />', () => {
     expect(getByTestId('dropdown-overlay').props.display).toBe('flex');
   });
 
+  it('should not open the dropdown when disabled', () => {
+    const { getByRole, getByTestId } = renderWithTheme(
+      <Dropdown>
+        <SelectInput label="Select Fruit" isDisabled />
+        <DropdownOverlay>
+          <ActionList>
+            <ActionListItem title="Mango" value="mango" />
+            <ActionListItem title="Apple" value="apple" />
+          </ActionList>
+        </DropdownOverlay>
+      </Dropdown>,
+    );
+
+    expect(getByTestId('dropdown-overlay').props.display).toBe('none');
```

Comment:
we shall not do test id unless absolutely necessary? the query by selector aren't helping in this case?

Reply:
There isn't any `.toBeVisible` equivalent in React Native that works as expected so I am checking the `display` prop.

Now the display prop is set on a `View` that doesn't have any accessibility role as such so to point to that exact div, we have to rely on getByTestId in React Native.

In Web, we can just get the ActionListItem element inside and check `toBeVisible` on it.